Integrations/Cybereason/Cybereason.py

```diff
@@ -1029,11 +1027,23 @@ def query_file_command():
             if 'companyName' in simple_values:
                 company_name = simple_values['companyName']['values'][0]
 
+            created_time = None
+            if 'createdTime' in simple_values:
+                created_time = timestamp_to_datestring(simple_values.get('createdTime', {}).get('values')[0])
```

Comment:
What will happen if `createdTime` is not in `simple_values` I think it could result in an exception no?.

Reply:
No need. It will not show this field if its not there.
Don't think an exception is needed